parts/linux/cloud-init/artifacts/init-aks-custom-cloud-mariner.sh

```diff
@@ -13,6 +13,8 @@ IFS=$IFS_backup
 
 cp /root/AzureCACertificates/*.crt /etc/pki/ca-trust/source/anchors/
 /usr/bin/update-ca-trust
+# Provide backwards compatibility with existing Ubuntu-based path of /etc/ssl/certs/ca-certificates.crt
+ln -s /etc/pki/tls/certs/ca-bundle.crt /etc/pki/tls/certs/ca-certificates.crt
```

Comment:
Both `/etc/pki/tls/certs/ca-certificates.crt` and `/etc/pki/tls/certs/ca-bundle.crt` should already be present on the host as they're both provided by the `ca-certificates-shared` package. 

If this isn't the case from observation on custom clouds we'll need to dig in a bit further as to why it's not there.